doc/starterkit/k4MarlinWrapperCLIC/Readme.md

```diff
@@ -156,23 +156,49 @@ k4run clicRec_e4h_input.py --EventDataSvc.input ttbar_edm4hep.root
 
 The ``MarlinDD4hep::InitializeDD4hep`` processor can be replaced by the ``k4SimGeant4::GeoSvc`` and the
 ``TrackingCellIDEncodingSvc`` the latter of which is part of the k4MarlinWrapper repository.
+This requires removing the wrapped `InitDD4hep` processor from the `algList` and adding `geoservice` and `cellIDSvc` to the `ExtSvc` in the `ApplicationMgr`.
 
-For example:
+We can create the `geoservice` and `cellIDSvc` with;
 
 ```python
 import os
 from Gaudi.Configuration import INFO
 from Configurables import GeoSvc, TrackingCellIDEncodingSvc
-svcList = []
+
 geoservice = GeoSvc("GeoSvc")
 geoservice.detectors = [os.environ["K4GEO"]+"/CLIC/compact/CLIC_o3_v15/CLIC_o3_v15.xml"]
 geoservice.OutputLevel = INFO
 geoservice.EnableGeant4Geo = False
-svcList.append(geoservice)
```

Comment:
It is more modular to append the service to the svcList here, because if I want to get rid of the service, I just have to comment the block here, and not find it again below, otherwise one might as well just put `ExtSvc = [geoservice, cellIDSvc, evtsvc]` in the ApplicationMgr.

Reply:
Ok, in that case would it need to happen in the same location as `evtsvc = k4DataSvc('EventDataSvc')` is being created, since that needs to go in the list first? (I assume they don't commute)

Reply:
That depends on what the "it" is that needs to happen?

But like this is:

https://github.com/key4hep/CLDConfig/blob/4cb3e121927352a36319910d95ef4cdcf5f8c483/CLDConfig/CLDReconstruction.py#L26-L30

Reply:
I think the `evtsvc` could in principle go anywhere since it should be independent of the other services. However, I would just put it first since it is (almost) always present and we should try to keep things consistent in our examples and documentation.

Reply:
I've rearranged things, is that what you had in mind?

Reply:
Yes, fine for me!
Thanks!